crates/flux-middle/src/lib.rs

```diff
@@ -76,6 +76,22 @@ pub static THEORY_FUNCS: LazyLock<UnordMap<Symbol, TheoryFunc>> = LazyLock::new(
     let bv_param0 = BvSize::Param(ParamSort::from(0));
     [
         // BitVector <-> int
+        TheoryFunc {
+            name: Symbol::intern("bv_zero_extend_32_to_64"),
+            fixpoint_name: Symbol::intern("app (_ zero_extend 32)"),
+            sort: rty::PolyFuncSort::new(
+                List::empty(),
+                rty::FuncSort::new(vec![BitVec(BvSize::Fixed(32))], BitVec(BvSize::Fixed(64))),
+            ),
+        },
```

Comment:
@ranjitjhala I think this is wrong. `(_ zero_extend 32)` *returns* a bv of size 32. The declaration should be

```rust
        TheoryFunc {
            name: Symbol::intern("bv_zero_extend_to_64"),
            fixpoint_name: Symbol::intern("app (_ zero_extend 64)"),
            sort: rty::PolyFuncSort::new(
                List::singleton(bv_param0),
                rty::FuncSort::new(vec![BitVec(bv_param0)], BitVec(BvSize::Fixed(64))),
            ),
        }
  ```
  
The test is working because `(_ bv2int 64)`, how `bv_bv64_to_int` is encoded as in z3, seems to ignore the `64`.